### ISSUE TITLE
Three suggestions

### ISSUE THREAD

**Issue by ghost** [fmhy/snowbin#13]
Hello, I am a user of this. But I have a couple of quality-of-life suggestions. I hope you'll take a look at them.

1 & 2: Change the mouse cursor to a pointer instead of a selector when selecting the theme and remove the underline from the links as they don't look the best.
![2024-11-24 22_15_29-snowbin - Chromium](https://github.com/user-attachments/assets/7b8b0841-9cac-41af-9c80-4484e9889db3)

3: Make the theme automatically switch based off of the OS theme.
![2024-11-24 22_18_01-snowbin - Brave](https://github.com/user-attachments/assets/bef64ca2-d415-40cc-adf0-bdec85d4fe20)

Thanks.

**Comment by taskylizard** [comment]
snowbin is currently undergoing a pending rewrite, which, unfortunately, is on hold. The issues you mentioned have already been addressed in my rewrite branch, except for the links. I agree they don't look good in the footer, which, at the time, was a design choice I made. That said, the only remaining piece to be rewritten is the backend infrastructure to make it independent of Cloudflare Workers.

I've been busy with other projects, and being the sole maintainer of snowbin and related projects has made progress more challenging. Hopefully, I can change this slow momentum someday.

**Comment by taskylizard** [comment]
Regarding 3: It should already work, but it may be blocked by your browser's ad blocker. It looks like you're using Brave, which might be blocking the `prefers-color-scheme` media query, supposedly for "privacy" reasons, like many other adblocking solutions (that I personally don't agree with).

https://github.com/taskyland/snowbin/blob/d27523f8c45371f2f97cf970e58b6d7ee4345f46/assets/static/theme.js#L3
